### ISSUE TITLE
Will_paginate code unused in profile.html.erb

### ISSUE THREAD

**Issue by Tlazypanda** [publiclab/plots2#8117]
### Please describe the problem (or idea)
Currently, the profile.html.erb file has `will_paginate` code to render tags but `@tags` is not available from `user_controller profile method` so it is not paginated. 

The resultant behaviour is that only some tags are shown without any pagination available.

**Comment by Tlazypanda** [comment]
@jywarren @cesswairimu can you kindly confirm this beahviour and also let me know should we add pagination to this or just eliminate this unused code? Thanks :v:

**Comment by Tlazypanda** [comment]
@jywarren @cesswairimu can you kindly review?  _Por favor_ :p Thanks :v:

**Comment by sagarpreet-chadha** [comment]
How does it look in both cases? Maybe that will help @jywarren @ebarry  to take decision in terms of UX.
Thanks 🎉

**Comment by cesswairimu** [comment]
@Tlazypanda you are right...seems it was replaced with `normal_tags` but was never updated

**Comment by KarishmaVanwari** [comment]
Is this issue open to work on? @cesswairimu @Tlazypanda
